.github/workflows/release.yaml

```diff
@@ -17,21 +17,21 @@ on:
 # Configure constants for this workflow.
 env:
   # The base filename of the binary produced by `cargo build`.
-  BINARY: bevy_quickstart
-  # The name to use for the packaged application produced by this workflow.
-  PACKAGE_NAME: bevy_quickstart
-  # The itch.io page to upload to, in the format: `user-name/project-name`.
-  # Comment this out to disable.
-  ITCH_TARGET: the-bevy-flock/bevy-quickstart
-  # The organization or author that owns the rights to the game.
-  OWNER: the-bevy-flock
+  cargo_binary_name: bevy_quickstart
```

Comment:
This sounds a bit like the name of the `cargo` binary itself now. Maybe `crate_binary_name` would be more accurate?

Reply:
Would `cargo_build_binary_name` be okay? I know it's verbose, but it's only in one place in the release workflow to move the binary produced by `cargo build` out of `target/...`. And it's only needed because it's currently not possible to tell `cargo` where to put the binary: https://github.com/rust-lang/cargo/issues/6790

Reply:
that's fine, yeah